index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Resume Builder</title>
+	</head>
+	<body>
+		<form action="#">
+			<fieldset class="basics" data-jsonSection="basics">
+				<label for="name">full name</label>
+				<input type="text" id="name" />
+				<label for="label">current/desired position</label>
+				<input type="text" id="label" />
+				<label for="picture">profile Picture</label>
+				<input type="image" id="picture" />
+				<label for="email">email</label>
```

Comment:
I also noted the schema for this was long. So first tried to attempt my goals/functions on a simple form. Then I could scale to this larger schema. 

Tip for future when taking on a big (relative) task like this. Start small.

Reply:
I got way too caught up in this markup I wish I had spent more time on the js side of things considering this was a js assignment